index.js

```diff
@@ -1,663 +1,246 @@
+    const [lower] = buttonObj[key][currentLang];
+    button.innerHTML = lower;
+    rowWraper.appendChild(button);
+  });
+  root.appendChild(rowWraper);
 });
 
 // language switching
 function switchLang() {
-    currentLang === 'en' ? currentLang = 'ru' : currentLang = 'en';
-    localStorage.setItem('lang', currentLang);
+  if (currentLang === 'en') {
+    currentLang = 'ru';
+  } else {
+    currentLang = 'en';
+  }
+  localStorage.setItem('lang', currentLang);
 }
 
 // Lowercase
 function updateKeys() {
-    keyboard.forEach(function(row) {
-        row.forEach(function(buttonObj) {
-            const key = Object.keys(buttonObj)[0];
-            const button = document.getElementById(key);
-            button.setAttribute('uniqueAttribute', buttonObj[key][currentLang][0]);
-            button.innerHTML = buttonObj[key][currentLang][0];
-        })
-    })
+  keyboard.forEach((row) => {
+    row.forEach((buttonObj) => {
+      const key = Object.keys(buttonObj)[0];
+      const button = document.getElementById(key);
+      button.setAttribute('uniqueAttribute', buttonObj[key][currentLang][0]);
+      const [lower] = buttonObj[key][currentLang];
+      button.innerHTML = lower;
+    });
+  });
 }
 
 // Uppercase
 function shiftUpdateKeys() {
-    keyboard.forEach(function(row) {
-        row.forEach(function(buttonObj) {
-            const key = Object.keys(buttonObj)[0];
-            const button = document.getElementById(key);
-            button.setAttribute('uniqueAttribute', buttonObj[key][currentLang][1]);
-            button.innerHTML = buttonObj[key][currentLang][1];
-        })
-    })
+  keyboard.forEach((row) => {
+    row.forEach((buttonObj) => {
+      const key = Object.keys(buttonObj)[0];
+      const button = document.getElementById(key);
+      button.setAttribute('uniqueAttribute', buttonObj[key][currentLang][1]);
+      const [, upper] = buttonObj[key][currentLang];
+      button.innerHTML = upper;
+    });
+  });
 }
 
 // Switching the state of the CapsLock button
-function capsUpdateKeys(){
-    cups === false ? (shiftUpdateKeys(), cups = true) : (updateKeys(), cups = false);
+function capsUpdateKeys() {
+  if (cups === false) {
+    shiftUpdateKeys();
+    cups = true;
+  } else {
+    updateKeys();
+    cups = false;
+  }
 }
 
-// Add mouse events 
-document.addEventListener('click', function(e) {
-
-    if (e.target.tagName === 'BUTTON'){
-
-        // if Backspace / Delete mouse
-        if (e.target.id === '8' || e.target.id === '46') {
-            input.value = input.value.slice(0, input.value.length - 1);
-            return;
-        }
-
-        //if Tab mouse
-        if(e.target.id === '9'){
-            const value = '\t';
-            input.value += value;
-            return;
-        }
+// Add mouse events
+document.addEventListener('click', (e) => {
+  if (e.target.tagName === 'BUTTON') {
+    // if Backspace / Delete mouse
+    if (e.target.id === '8' || e.target.id === '46') {
```

Comment:
you could've stored key titles in your data map, then it'd be obvious what you're doing in these checks.

Reply:
like keys.backspace instead of '8'